### PR TITLE
fix(AutoComplete): broken AutoComplete in certain case

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

### CODE REVIEW

File: src/components/forms/auto-complete/RuiAutoComplete.spec.ts [hunk 1]
```diff
@@ -122,7 +122,27 @@ describe('autocomplete', () => {
...
+    expect(wrapper.emitted().input!.at(-1)![0]).toEqual(null);
+
+    // doesn't break when use chips
+    await wrapper.setProps({
+      chips: true,
+    });
+
+    await nextTick();
+
+    expect(wrapper.html()).toBeTruthy();
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply src/components/forms/auto-complete/RuiAutoComplete.spec.ts:145]: has content and doesn't break

[the remark this answers is not in the document]
Comment (kelsos) [reply src/components/forms/auto-complete/RuiAutoComplete.spec.ts:145]: Ok so when the bug was happening wrapper would be empty then?

[the remark this answers is not in the document]
Comment (lukicenturi) [reply src/components/forms/auto-complete/RuiAutoComplete.spec.ts:145]: okay apparently I don't need to do that, the test will fail if that error is found, I will remove this part

**Review by kelsos — APPROVED** [review]
lgtm thanks :)

**Pull request merged**